Documentation/project-docs/platform-names.md

```diff
@@ -8,19 +8,19 @@ This will also list some aliases. As you'll see the aliases aren't always correc
 
 **Also referred to as**: Desktop, full framework, in-box framework
 
-This refers to the .NET Framework that shipped in 2002. It's the main framework folks target today and allows you to build a wide variety of applications, such as WinForms, WPF, ASP.NET, and command line tools.
+This refers to the .NET Framework that first shipped in 2002 and is now up to version 4.6.1. It's the main framework folks target today and allows you to build a wide variety of applications, such as WinForms, WPF, ASP.NET, and command line tools.
 
-The .NET Framework was designed to run on Windows only. Some versions of the .NET Framework come pre-installed with Windows, some require to be installed. However, in both cases the .NET Framework is a system-wide component.
+The .NET Framework was designed to run on Windows only. Some versions of the .NET Framework come pre-installed with Windows, some require to be installed. However, in both cases the .NET Framework is a system-wide component. Application do not include .NET Framework DLLs when deploying; the correct .NET version must be on the machine.
 
 ## .NET Core
 
 **Also referred to as**: UWP, ~~Store~~
 
-Originally, .NET Core was the identifier we used to describe the surface area Windows 8 store applications can use. When we designed the API set, we wanted to create a foundation for .NET where portability is a first class concern for the layering and componentization. For more details, read [this blog post](http://blogs.msdn.com/b/dotnet/archive/2014/12/04/introducing-net-core.aspx).
+Originally, .NET Core was the identifier we used to describe the .NET APIs Windows 8 store applications could use. When we designed the API set, we wanted to create a foundation for .NET where portability is a first class concern for the layering and componentization. For more details, read [this blog post](http://blogs.msdn.com/b/dotnet/archive/2014/12/04/introducing-net-core.aspx).
 
-Today, .NET Core no longer only powers store applications. .NET Core is the name for the open source, cross-platform stack that ASP.NET Core and UWP applications are built on. The stack includes a set of framework libraries (CoreFX), a JIT based runtime (CoreCLR), an AOT based runtime (CoreRT), and a set of tooling (such as the dotnet CLI).
+Today, .NET Core no longer just for store applications. .NET Core is the name for the open source, cross-platform stack that ASP.NET Core and UWP applications are built on. The stack includes a set of framework libraries (CoreFX), a JIT based runtime (CoreCLR), an AOT based runtime (CoreRT), and a set of tooling (such as the dotnet CLI).
```

Comment:
I don't know if this is clearer, personally. Also, it should be ".NET Core **is** no longer just for store applications" (or something similar).